GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -177,14 +177,14 @@ protected virtual bool TryRunGVFSWithArgs(string args, out string consoleError)
 
         private bool IsGVFSUpgradeAllowed(out string consoleError)
         {
-            bool isConfirmed = string.Equals(this.CommandToRerun, GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm, StringComparison.OrdinalIgnoreCase);
+            bool isConfirmed = string.Equals(this.CommandToRerun, GVFSPlatform.Instance.Constants.UpgradeConfirmCommandMessage, StringComparison.OrdinalIgnoreCase);
             string adviceText = null;
             if (!this.IsElevated())
             {
-                adviceText = isConfirmed ? $"Run {this.CommandToRerun} again from an elevated command prompt." : $"To install, run {GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm} from an elevated command prompt.";

```

Comment:
Can we simplify this to just instruct the user to re-run the command elevated?

`adviceText = GVFS.Platform.Instance.Constants.RunCommandElevated`, where we can say "Run the command from an elevated command prompt" or "Run the command as root (or with sudo)" on Mac...